scripts/release.py

```diff
@@ -32,7 +32,16 @@
 
 JIRA_RE = re.compile(r"\b[A-Z]{2,}-\d+\b")
 PR_BRANCH_RE = re.compile(r"remotes/edx/pr/(\d+)")
-PROJECT_ROOT = path(__file__).abspath().dirname()
+
+
+def project_root():
```

Comment:
The old code worked, but updating to the latest version of GitPython broke it if you were running from inside the `scripts` directory. I assume that the old version of GitPython implicitly did directory traversal to find the git repository, and the new version does not.